frontend/packages/dev-console/src/components/pipelines/PipelineVisualizationTask.tsx

```diff
@@ -117,21 +116,29 @@ export const PipelineVisualizationTask: React.FC<PipelineVisualizationTaskProp>
 };
 const TaskComponent: React.FC<TaskProps> = ({ task, status, name, isPipelineRun }) => {
   const taskData = task.data;
+  const stepList = _.get(taskData, ['spec', 'steps'], []);
+  const showStatusState: boolean = isPipelineRun && !!status && !!status.reason;
+
   return (
     <li className={cx('odc-pipeline-vis-task')}>
       <div className="odc-pipeline-vis-task__connector" />
       <Tooltip
         position="bottom"
         enableFlip={false}
-        content={
-          <PipelineVisualizationStepList steps={(taskData.spec && taskData.spec.steps) || []} />
-        }
+        content={<PipelineVisualizationStepList steps={stepList} />}
       >
         <div className="odc-pipeline-vis-task__content">
-          <div className={cx('odc-pipeline-vis-task__title', { 'is-text-center': !isPipelineRun })}>
-            {name || _.get(task, ['metadata', 'name'], '')}
+          <div
+            className={cx('odc-pipeline-vis-task__title-wrapper', {
+              'is-text-center': !isPipelineRun,
+            })}
+          >
+            <div className="odc-pipeline-vis-task__title">
+              {name || _.get(task, ['metadata', 'name'], '')}
+            </div>
+            {showStatusState && <TaskComponentTaskStatus steps={stepList} status={status} />}
           </div>
-          {isPipelineRun && status && status.reason && (
+          {isPipelineRun && (
```

Comment:
Fixed the issue found in https://jira.coreos.com/browse/ODC-1834 by reserving the space

Reply:
![ReserveSpace](https://user-images.githubusercontent.com/8126518/64713076-46ce2080-d48a-11e9-83c3-99e81de145b4.gif)